.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+If reporting an issue please try to provide the information asked below.
+
+Before reporting an issue please:
+
+1. make sure that you're using the latest published GVM components for the
+   release you are using: http://openvas.org/install-source.html.
+2. check the list of issues whether it isn't already reported.
+3. be aware that this is not a support forum. If your issue is more a question
+   than a bug report, please check http://openvas.org/ for available support
+   options.
+-->
+
+### Expected behavior
+
+### Current behavior
+
+### Steps to reproduce
+1.
+2.
+3.
+
+### GVM versions
+**gsa:** (gsad --version)
+
+**gvm:** (gvmd --version)
+
+**openvas-scanner:** (openvassd --version)
+
+**gvm-libs:**
+
+**gvm-tools:** (gvm-cli --version)
+
+### Environment
+**Operating system:**
+
+**Installation method / source:** (packages, source installation)
+
+### Logfiles
```

Comment:
It might be useful to describe where the logfiles can be found and/or how the application can be configured to produce debug output.